File: Problem 7.py
n = 10001
counter = 0 
prime = 0
i = 1

def checkPrime(n: int):
    if (n == 1): return False
    i = 2
    while (i*i <= n):
        if (n%i == 0):
            return False
        else: i+=1
    return True
    
while (counter != n):
    if checkPrime(i):
        prime = i
        counter+=1
        i+=1
    else: i+=1

print(prime)

File: Problem 7.js
primes =  [2]
counter = 3

while(primes.length<=3){
	primeCheck(counter)
}
console.log(primes)

function primeCheck(n){
	let flag = 1
	for (let prime in primes){
		flag = 0
		if (n%prime == 0){
			flag+=2
		}
	}
	if (flag == 1){
		primes.push(n)
		counter++
	}
}